entrypoint.sh

```diff
@@ -264,7 +264,7 @@ test_pkg() {
 
     ulimit -c unlimited
     testbin="./${pkg}/${pkg##*/}.test"
-    show go test -v "${testargs[@]}" -o "${testbin}" "./${pkg}"
+    show go test -timeout 15m -v "${testargs[@]}" -o "${testbin}" "./${pkg}"
```

Comment:
Is this always the case that we hit the timeout with these additional tests?

Reply:
Yes, running all 3 new tests causes the rbd run to take just over 10m. Note that rbd is the only package that takes this long, but I am far too lazy to try and customize the timeout only for rbd.